<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>elevator</class>
 <widget class="QWidget" name="elevator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="btn_call">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>30</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>报警</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_door">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>361</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>开/关 门</string>
   </property>
   <widget class="QPushButton" name="btn_open">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>开门</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_close">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>30</y>
      <width>71</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>关门</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_door_status">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>111</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;门的状态&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_floor">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>楼层</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_message">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
   <property name="title">
    <string>当前楼层</string>
   </property>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>72</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_current_floor">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>51</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;层数&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_door</zorder>
  <zorder>btn_call</zorder>
  <zorder>groupBox_floor</zorder>
  <zorder>groupBox_message</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
